handlers/rsa.go

```diff
@@ -70,7 +60,26 @@ func (rh *RsaHandler) GetName() string {
 // Init initializes the handler.
 // Takes one argument for persistent data directory.
 func (rh *RsaHandler) Init(opts ...string) (err error) {
+	if len(opts) != 2 {
+		return fmt.Errorf("RSA Init received %d items in opts", len(opts))
```

Comment:
Let's provide some error message that helps the command line user to understand what parameter is expected
		`return fmt.Errorf("rsa handler requires a file path with the validation RSA key")`